src/main/scala/mesosphere/marathon/MarathonSchedulerActor.scala

```diff
@@ -452,9 +452,9 @@ class SchedulerActions(
     * Make sure the runSpec is running the correct number of instances
     */
   // FIXME: extract computation into a function that can be easily tested
-  def scale(runSpec: RunSpec): Unit = {
+  def scale(runSpec: RunSpec): Future[Unit] = async {
```

Comment:
The scapegoat analysis tool does not accept accept `asyn/await` (problem in scapegoat) Please add a ` @SuppressWarnings` annotation. (Check with `sbt scapegoat`)